### PR TITLE
Fix/emit nist goes

### PR DESCRIPTION
**Pull request by amarouane-ABDELHAK**
Fixing slash

### CODE REVIEW

File: .github/workflows/deploy-goes.yml [hunk 1]
```diff
@@ -19,6 +19,8 @@ jobs:
         run: |
           if [ "${{ github.ref }}" = "refs/heads/main" ]; then
             echo "env_name=staging" >> $GITHUB_OUTPUT
+          elif [ "${{ github.ref }}" = "refs/heads/deploy-goes-to-prod" ]; then
+            echo "env_name=production" >> $GITHUB_OUTPUT
```
Comment (slesaad) [inline .github/workflows/deploy-goes.yml:23]: time to remove?

File: .github/workflows/pr-preview.yml [hunk 2]
```diff
@@ -156,46 +161,39 @@ jobs:
...
-              echo REACT_APP_BASE_PATH="${{vars.REACT_APP_BASE_PATH_GOES}}" >> ./goes-plume-viewer/.env
-              echo REACT_APP_PUBLIC_URL="${GOES_PATH}" >> ./goes-plume-viewer/.env
-
-              URL_VALUE=$(grep "REACT_APP_PUBLIC_URL" ./goes-plume-viewer/.env)
-              echo "The value of REACT_APP_PUBLIC_URL is: ${URL_VALUE}"
-
+              echo REACT_APP_MAPBOX_TOKEN="${{ secrets.REACT_APP_MAPBOX_ACCESS_TOKEN }}" >> ./goes-plume-viewer/.env
+              echo REACT_APP_MAPBOX_STYLE_URL="${{ secrets.REACT_APP_MAPBOX_STYLE_URL }}" >> ./goes-plume-viewer/.env
+              echo REACT_APP_BASE_PATH="${{ vars.REACT_APP_BASE_PATH_GOES }}/${{ env.PR_STRING }}" >> ./goes-plume-viewer/.env
+              echo PUBLIC_URL="${{ vars.REACT_APP_BASE_PATH_GOES }}/${{ env.PR_STRING }}" >> ./goes-plume-viewer/.env
```
Comment (slesaad) [inline .github/workflows/pr-preview.yml:185]: do we need both of these?

File: emit-ch4plume-v1/package.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "deploy": ". ./.env; webpack --mode=${STAGE:-development} --env PUBLIC_URL=${PUBLIC_URL}",
+    "deploy": ". ./.env; webpack --mode=${STAGE:-development} --env PUBLIC_URL=${PUBLIC_URL}; cp -r data dist",
```
Comment (slesaad) [inline emit-ch4plume-v1/package.json:8]: this change was already merged in another PR

File: emit-ch4plume-v1/index.html [hunk 4]
```diff
@@ -19,7 +19,6 @@
     <script src="https://code.jquery.com/jquery-1.9.1.js"></script>
     <script src="https://code.jquery.com/ui/1.10.2/jquery-ui.js"></script>
```
Comment (slesaad) [inline emit-ch4plume-v1/index.html:21]: this change was already merged in another PR

File: .github/workflows/pr-preview.yml [hunk 5]
```diff
@@ -14,7 +14,8 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  S3_BUCKET: ghgc-custom-interfaces-develop
+  S3_BUCKET: ghgc-dev-custom-interfaces
```
Comment (slesaad) [inline .github/workflows/pr-preview.yml:17]: can we please keep using the same bucket? (it follows the same convention for staging and production buckets too, so don't want to be confused later on)
just do the work needed to update cloudfront to have access to the bucket

File: nist-interface/.env [hunk 6]
```diff
@@ -1 +0,0 @@
-REACT_APP_FEATURES_API_URL="https://dev.ghg.center/api/features"
```
[the remark this answers is not in the document]
Comment (amarouane-ABDELHAK) [reply nist-interface/.env:1]: I don't like to commit `.env` file. Will add that to actions.yml instead
Looks like the preview works without it

File: .github/workflows/deploy-goes.yml (hunk 1 above)
Comment (amarouane-ABDELHAK) [reply .github/workflows/deploy-goes.yml:23]: Yes

Comment (slesaad) [reply .github/workflows/deploy-goes.yml:23]: why no remove? lol

**Review by slesaad — APPROVED** [review]

**Pull request merged**